src/server/pps/server/api_server.go

```diff
@@ -2841,6 +2842,74 @@ func (a *apiServer) RunPipeline(ctx context.Context, request *pps.RunPipelineReq
 	return &types.Empty{}, nil
 }
 
+func (a *apiServer) RunCron(ctx context.Context, request *pps.RunCronRequest) (response *types.Empty, retErr error) {
+	func() { a.Log(request, nil, nil, 0) }()
+	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
+
+	pachClient := a.env.GetPachClient(ctx)
+	ctx = pachClient.Ctx() // pachClient will propagate auth info
```

Comment:
Any reason to declare this context? I think it's unused throughout the function. (I suspect this was just copy-pasted from elsewhere.)